src/scaffoldfitter/fitter.py

```diff
@@ -852,6 +853,41 @@ def getDataRMSAndMaximumProjectionErrorForGroup(self, groupName):
 
         return None, None
 
+    def getLowestElementJacobian(self, mesh_group=None):
+        """
+        Get the information on the 3D element with the worst jacobian value (most negative).
+        For a right-handed element values <= 0.0 are bad, the opposite holds for left-handed
+        elements.
+        Optional mesh group parameter allows the user to make the calculation over a different group
+        from the whole mesh.
+        :param mesh_group: Optional parameter to specify a particular mesh group to make the calculation over.
+        :return: Element identifier, minimum jacobian value. Values are -1, inf if there is no data or bad fields.
+        """
+        with ChangeManager(self._fieldmodule):
+            jacobian = create_xi_reference_jacobian_determinant_field(self._modelCoordinatesField)
```

Comment:
needs reference coordinate field argument:
jacobian = create_jacobian_determinant_field(self._modelCoordinatesField, self._modelReferenceCoordinatesField)
also do a test evaluation in any test. in test_fitcube.py test_alignMarkersFitRegularData